packages/base/Step/src/Stepper/Stepper.tsx

```diff
@@ -44,48 +36,40 @@ const Stepper = forwardRef<HTMLDivElement, Props>((props, ref) => {
     overflowEllipsis = false,
     ...rest
   } = props
-  const classes = useStyles()
 
   return (
-    <MUIStepper
-      {...rest}
-      ref={ref}
-      activeStep={active}
-      connector={<StepConnector direction={direction} />}
-      className={cx(classes.root, className)}
+    <div
+      className={cx('flex', className, {
+        'flex-row items-center gap-1': direction === 'horizontal',
+        'flex-col': direction === 'vertical',
+      })}
```

Comment:
I believe we should use `twMerge` here, as we allow `className` customizations, I also move `className` to have a higher priority over other class names
```suggestion
      className={txMerge('flex', cx({
        'flex-row items-center gap-1': direction === 'horizontal',
        'flex-col': direction === 'vertical',
      }), className)}
```

Reply:
if we merge with className that comes as a prop, then yes, we should use `twMerge` helper